index.bs

```diff
@@ -338,21 +338,31 @@ So, if the object inherits from {{EventTarget}}, add a corresponding
 [=event handler IDL attributes=] on the {{GlobalEventHandlers}} interface, instead of directly on
 the relevant element interface(s).</p>
 
+<h2 id="event-design">Event Design</h2>
+
+<h3 id="events-are-for-notification">Events are for notification</h3>
+
+Try to design DOM events to deliver after-the-fact notifications of changes. It may be tempting to try to trigger side-effects from the action of <code>dispatchEvent</code>, but in general this is <a href="https://lists.w3.org/Archives/Public/public-webapps/2014AprJun/0510.html">discouraged</a>.
```

Comment:
It's more than discouraged as it requires changing the DOM Standard each time it's done.